docker/skopeo_runner/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM fedora:33
+
+# TODO: should the image contain any special labels?
+
+WORKDIR /src
+
+RUN dnf install -y skopeo
```

Comment:
in docker images, I prefer to do the installation & clean up in one layer.

Reply:
Added